Index: vtysh/vtysh_main.c
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vtysh/vtysh_main.c b/vtysh/vtysh_main.c
--- a/vtysh/vtysh_main.c	(revision HEAD)
+++ b/vtysh/vtysh_main.c	(revision Staged)
@@ -42,6 +42,7 @@

 #include "vtysh/vtysh.h"
 #include "vtysh/vtysh_user.h"
+#include "vtysh/vtysh_extensions.h"

 /* VTY shell program name. */
 char *progname;
